### DISCUSSION TITLE
Why is Xournal mobile not available on Playstore?

### DISCUSSION THREAD

**Discussion by gybeom8008** [xournalpp/xournalpp#5570] [Q&A] [answered]
I'm not sure if this is the right place to ask... but when I follow the Google Play link, it returns 'Not found' page. Is it temporary?

![Screenshot from 2024-03-17 21-05-02](https://github.com/xournalpp/xournalpp/assets/46715731/9549450f-7ebc-4a07-8179-1f023cb149d9)

**Comment by rolandlo** [comment, marked as answer]
The short answer: xournalpp mobile has been unmaintained for a couple of years now, see https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/issues/56
